### ISSUE TITLE
Introduce tests

### ISSUE THREAD

**Issue by digiwizkid** [sschueller/peertube-android#222]
Hi!
Is there any plan to introduce tests (Unit/ Integration)?

**Comment by sschueller** [comment]
Yes, definitely. However I have not gotten around to it. If anyone wants to get started I would appreciate it.
